app/Http/Controllers/Auth/LoginController.php

```diff
@@ -110,7 +110,9 @@ protected function attemptSecondaryAuth()
         } else {
             Auth::login(Auth::guard('vatsim-sso')->user(), true);
 
-            return redirect('/');
+            $intended = Session::get('url.intended') ?? route('site.home');
```

Comment:
```suggestion
            $intended = Session::pull('url.intended', route('site.home'));
```

Also suggest moving this up to the top of the method and removing both existing `$intended` variables because its always used. The above also eliminates the need for the `Session::remove`